Vector/ViewController/HomeViewController.m

```diff
@@ -205,14 +205,6 @@ - (void)viewDidAppear:(BOOL)animated
     }
 }
 
-- (void)viewWillDisappear:(BOOL)animated
-{
-    [super viewWillDisappear:animated];
-
-    _selectedRoomId = nil;
-    _selectedRoomSession = nil;
-}
```

Comment:
It was used to unselect a room but this PR implements the unselection with another mean that works better.
This reset looses the selected room id which prevents from being able to do room selection on viewWillAppear.
Last argument, it is better to manage this in one dedicated place: [self closeSelectedRoom].